baselines/FedStar/FedStar/server.py

```diff
@@ -0,0 +1,219 @@
+##########################################
+import os
+import sys
+
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
+os.environ["GRPC_VERBOSITY"] = "ERROR"
+GRPC_MAX_MESSAGE_LENGTH: int = 536_870_912
+##########################################
+import argparse
+import pathlib
```

Comment:
```suggestion
```

Reply:
Not used, then remove.

Reply:
Thanks for the suggestion, I have removed the imports.